src/kafka-util/src/admin.rs

```diff
@@ -41,6 +41,19 @@ pub async fn create_topic<'a, C>(
     admin_opts: &AdminOptions,
     new_topic: &'a NewTopic<'a>,
 ) -> Result<(), CreateTopicError>
+where
+    C: ClientContext,
+{
+    create_topic_allow_existing(client, admin_opts, new_topic, true).await
+}
+
+/// Like `create_topic` but requires specification of whether topic can exist.
+pub async fn create_topic_allow_existing<'a, C>(
+    client: &'a AdminClient<C>,
+    admin_opts: &AdminOptions,
+    new_topic: &'a NewTopic<'a>,
+    allow_existing: bool,
```

Comment:
Boolean parameters make me twitchy. I'm have tempted to say the public interface of this crate should expose two separate methods (`ensure_topic` and `create_new_topic` or something) but that makes your life difficult in `coord` since you've already got a `succeed_if_exists: bool` parameter. Oh well.

Reply:
ha yeah I feel the same way and went through basically the same thought process